docs/en/getting-started/get-started-stack.asciidoc

```diff
@@ -43,18 +40,33 @@ data, such as logs or decoded network packets.
 
 [TIP]
 ==========
-You can run Elasticsearch on your own hardware, or use our
-https://www.elastic.co/cloud/elasticsearch-service[hosted Elasticsearch Service]
-on Elastic Cloud. The Elasticsearch Service is available on both AWS and GCP.
+You can run {es} on your own hardware, or use our
+https://www.elastic.co/cloud/elasticsearch-service[hosted {es} Service]
+on Elastic Cloud. The {es} Service is available on both AWS and GCP.
 {ess-trial}[Try out the
-Elasticsearch Service for free].
+{es} Service for free].
 ==========
 
 To download and install {es}, open a terminal window and use the commands that
-work with your system (<<deb, deb>> for Debian/Ubuntu, <<rpm, rpm>> for
-Redhat/Centos/Fedora, <<mac, mac>> or <<brew, brew>> for OS X, <<linux, linux>>
-for Linux, and <<win, win>> for Windows):
+work with your system:
+
+* <<deb, deb>> for Debian/Ubuntu
+* <<rpm, rpm>> for Redhat/Centos/Fedora
+* <<mac, mac>> or <<brew, brew>> for OS X
+* <<linux, linux>> for Linux
+* <<win, win>> for Windows
 
+When you start {es} for the first time, the following security configuration
```

Comment:
maybe a sentence that is more user friendly, similar to
```suggestion
When you start {es} for the first time, security features such as authentication, authorization and network encryption (TLS) for elasticsearch are enabled by default. The following security configuration
```
? Think it's important we tell users _what_ we do for them as well as how